compiler/codegen/src/compile.rs

```diff
@@ -709,6 +709,7 @@ impl Compiler {
                 orelse,
                 finalbody,
             } => self.compile_try_statement(body, handlers, orelse, finalbody)?,
+            TryStar { .. } => unimplemented!("except*"),
```

Comment:
if you could change this into a simple call to `compile_try_star` which unconditionally raises an exception (basically a copy paste of `compile_match`. This way the interpreter won't be brought down if it encounters it someplace (e.g a test file we update).

Reply:
On it